README.md

```diff
@@ -131,6 +131,15 @@ Or
 oc adm must-gather --image=quay.io/kubevirt/must-gather -- PROS=3 /usr/bin/gather_images
 ```
 
+### Targeted gathering - Instancetypes and preferences
+
+Instancetypes and preferences are not currently collected by default but can be
+by running the following command:
+
+```sh
+oc adm must-gather --image=quay.io/kubevirt/must-gather -- /usr/bin/gather_instancetypes
```

Comment:
@machadovilaca - I think this should be re-format after #158 was merged, right? Could you offer the new format?

Reply:
Ah, rebased and updated.